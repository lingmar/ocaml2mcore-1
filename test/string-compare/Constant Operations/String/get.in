let s = "hello" in
String.get s 3
