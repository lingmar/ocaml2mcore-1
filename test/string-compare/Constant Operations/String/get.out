mexpr
let s = "hello" in
get s 3
